language_service/src/completion.rs

```diff
@@ -131,6 +139,11 @@ fn collect_hardcoded_words(expected: WordKinds) -> Vec<Completion> {
                 completions.extend([
                     Completion::new("EntryPoint".to_string(), CompletionItemKind::Interface),
                     Completion::new("Config".to_string(), CompletionItemKind::Interface),
+                    Completion::new("Unimplemented".to_string(), CompletionItemKind::Interface),
```

Comment:
Do want people to know about `Unimplemented`? Given the uses for it, I wonder if it would be better to leave it out. We introduced it in case we want to have more specific errors for library functions we intentionally haven't implemented, but we never actually used it anywhere.